README.md

```diff
@@ -6,11 +6,53 @@ Automatically store Device and FCM Token information for Firebase Auth Users in
 
 ## Installation
 
-TODO
+Add `FirebaseDeviceStore` to your `Podfile`:
+
+```
+pod 'FirebaseDeviceStore'
+```
+
+### Objective-C
+
+Make sure you have `use_frameworks!` enabled in your `Podfile`
 
 ## Example usage
 
-TODO
+### Swift
+
+Import the library:
+
+```
+import FirebaseDeviceStore
+```
+
+Initialise the library:
+
+```
+let deviceStore = FirebaseDeviceStore.init(app: FirebaseApp.app()!);
```

Comment:
FWIW, I think 99% of people will only have the default FirebaseApp and so perhaps we should make app optional and allow `.init()`?

Reply:
That makes sense, I've made the change here and on Android